internal/server/singleprocess/state/app_operation.go

```diff
@@ -687,6 +717,7 @@ type listOperationsOptions struct {
 	Status        []*pb.StatusFilter
 	Order         *pb.OperationOrder
 	PhysicalState pb.Operation_PhysicalState
+	WatchSet      memdb.WatchSet
```

Comment:
Would love to see tests around this use case for both the start and complete time indexes.

Reply:
Donzo!